### PR TITLE
UPD readme

### PR DESCRIPTION
**Pull request by ffuuugor**
Link tutorials, talks and migration guide on the main readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,14 @@
 
 [![CircleCI](https://circleci.com/gh/pytorch/opacus.svg?style=svg)](https://circleci.com/gh/pytorch/opacus)
 
-[Opacus](https://opacus.ai) is a library that enables training PyTorch models with differential privacy. It supports training with minimal code changes required on the client, has little impact on training performance and allows the client to online track the privacy budget expended at any given moment.
+[Opacus](https://opacus.ai) is a library that enables training PyTorch models with differential privacy. 
+It supports training with minimal code changes required on the client, has little impact on 
+training performance and allows the client to online track the privacy budget expended at any given moment.
```
Comment (karthikprasad) [inline README.md:9]: I should really stop being annoying with these nits!

```suggestion
training performance, and allows the client to online track the privacy budget expended at any given moment.
```

File: README.md [hunk 2]
```diff
@@ -48,6 +50,36 @@ model, optimizer, data_loader = privacy_engine.make_private(
 
 The [MNIST example](https://github.com/pytorch/opacus/tree/main/examples/mnist.py) shows an end-to-end run using opacus. The [examples](https://github.com/pytorch/opacus/tree/main/examples/) folder contains more such examples.
 
+### Migrating to 1.0
+
+With the recent release we've introduced many improvements to the library, but also some breaking changes.
```
Comment (karthikprasad) [inline README.md:55]:
```suggestion
Opacus 1.0 introduced many improvements to the library, but also some breaking changes.
```

File: README.md [hunk 3]
```diff
@@ -48,6 +50,36 @@ model, optimizer, data_loader = privacy_engine.make_private(
...
+If you've been using opacus before and want to update to the latest release,
```
Comment (karthikprasad) [inline README.md:56]:
```suggestion
If you've been using Opacus 0.x and want to update to the latest release, 
```

File: README.md [hunk 4]
```diff
@@ -48,6 +50,36 @@ model, optimizer, data_loader = privacy_engine.make_private(
...
+- [Introduction to advanced features of Opacus](https://github.com/pytorch/opacus/blob/main/tutorials/intro_to_advanced_features.ipynb)
+
+### Blogposts and talks
+
+If you want to learn more about DP-SGD and related topics, check our our series of blogposts and talks:
+
+- [Differential Privacy Series Part 1 | DP-SGD Algorithm Explained](https://medium.com/pytorch/differential-privacy-series-part-1-dp-sgd-algorithm-explained-12512c3959a3)
+- [Differential Privacy Series Part 2 | Efficient Per-Sample Gradient Computation in Opacus](https://medium.com/pytorch/differential-privacy-series-part-2-efficient-per-sample-gradient-computation-in-opacus-5bf4031d9e22)
+- [PriCon 2020 Tutorial: Differentially Private Model Training with Opacus](https://www.youtube.com/watch?v=MWPwofiQMdE&list=PLUNOsx6Az_ZGKQd_p4StdZRFQkCBwnaY6&index=52)
+- [Differential Privacy on PyTorch | PyTorch Developer Day 2020](https://www.youtube.com/watch?v=l6fbl2CBnq0)
```
Comment (karthikprasad) [inline README.md:79]: Add the link to 2021 talk instead (or at least in addition to)?

File: README.md [hunk 5]
```diff
@@ -48,6 +50,36 @@ model, optimizer, data_loader = privacy_engine.make_private(
...
+
+
```
Comment (karthikprasad) [inline README.md:81]:
```suggestion
```

**Review by karthikprasad — APPROVED** [review]

**Comment by ffuuugor** [comment]
>I should really stop being annoying with these nits!

Please continue!

File: README.md [hunk 6]
```diff
@@ -48,6 +50,35 @@ model, optimizer, data_loader = privacy_engine.make_private(
...
+Opacus 1.0 introduced many improvements to the library, but also some breaking changes.
+If you've been using Opacus 0.x and want to update to the latest release, 
+please use this [Migration Guide](https://github.com/pytorch/opacus/blob/main/Migration_Guide.md)
+
+
+## Learn more
+
+### Interactive tutorials
+
+We've build a series of IPython-based tutorials as a gentle introduction to training models
```
Comment (karthikprasad) [inline README.md:64]:
```suggestion
We've built a series of IPython-based tutorials as a gentle introduction to training models
```

**Pull request merged**